packages/react-bootstrap-table2/test/header-cell.test.js

```diff
@@ -286,5 +286,65 @@ describe('HeaderCell', () => {
         expect(classesCallBack.calledWith(column, index)).toBe(true);
       });
     });
+
+    describe('when column.headerAttrs prop is defined', () => {
+      beforeEach(() => {
+        column = {
+          dataField: 'id',
+          text: 'ID',
+          headerAttrs: {
+            title: 'title',
+            'data-test': 'test'
+          }
+        };
+      });
+
+      describe('when headerAttrs is an object', () => {
+        it('should render column.headerAttrs correctly', () => {
+          wrapper = shallow(<HeaderCell column={ column } index={ index } />);
+
+          expect(wrapper.length).toBe(1);
+          expect(wrapper.find('th').prop('data-test')).toEqual(column.headerAttrs['data-test']);
+          expect(wrapper.find('th').prop('title')).toEqual(column.headerAttrs.title);
+        });
+
+        describe('when column.headerTitle prop is defined', () => {
```

Comment:
Nice case, but we should cover all the overwrite case when project is still small.

```js
describe('when column.headerTitle, column.style, column.classes, etc  prop is defined', () => {
 
  //..  put all overwrite case here be each different ""it""

});
```